### ISSUE TITLE
pxrl* should be able to use regex

### ISSUE THREAD

**Issue by josephwb** [FePhyFoFum/phyx#59]
Would be very convenient to be able to use regex when relabelling alignments (`pxrls`) or trees (`pxrlt`). ATM I am writing ("bad") labels to file, performing `sed` on them (to make "good" names), and using phyx to relabel.

**Comment by josephwb** [comment]
e.g. to relabel all files (with different sampling) by removing a suffix:
```
#!/bin/bash
for i in $( ls *.phy); do
    pxlssq -l -s $i -o bad_names.txt
    sed -- 's/\..*//g' bad_names.txt > good_names.txt
    pxrls -c bad_names.txt -n good_names.txt -s $i -o $i.relabelled
done
```
Works but not very elegant.

**Comment by josephwb** [comment]
`pxlstr` is done (3b51099f196d9f8f82a135eb1d2f12458b019a2d). Use the same approach for sequences (`pxrms`).

**Comment by josephwb** [comment]
Ok this too is implemented (with 1c46b0d0b0eefad9837837594774431fc41f3c4c). Will close this when merging to master.

**Comment by josephwb** [comment]
Doh the above commit deals with `pxrms`, not `pxrls`. This issue is still wide open. Will get to it in a bit.

**Comment by josephwb** [comment]
`pxrlt` addressed with 46e6311027e25f3905d56b6d6daf0e713e90981c, and `pxrls` with 58b71b617c3cbddf27a3425eb48ca0fc439ea449 ... four years later...
